coins.json

```diff
@@ -322,4 +333,37 @@
 	"bitcore": [
 		"https://2x-bitcore4.trezor.io"
 	]
-}]
+},
+{
+	"coin_name": "Bitcoin Gold",
+	"coin_shortcut": "BTG",
+	"coin_label": "Bitcoin Gold",
+	"address_type": 38,
+	"address_type_p2sh": 23,
+	"maxfee_kb": 500000,
+	"minfee_kb": 1000,
+	"signed_message_header": "Bitcoin Gold Signed Message:\n",
+	"hash_genesis_block": "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f",
+	"xprv_magic": "0488ade4",
+	"xpub_magic": "0488b21e",
+	"xpub_magic_segwit_p2sh": "049d7cb2",
+	"bech32_prefix": null,
+	"bip44": 156,
+	"segwit": true,
+	"forkid": 79,
+	"force_bip143": false,
```

Comment:
Update: https://github.com/BTCGPU/BTCGPU/pull/83#issuecomment-342559904

Reply:
Does the last update this PR is ready to be merged?

Reply:
> will merge the full BIP143 style SIGHASH version very soon

I think they are going to use BIP143 signatures for non-SegWit inputs